src/main/java/seedu/address/logic/commands/DeleteEventCommand.java

```diff
@@ -0,0 +1,62 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.event.Event;
+
+/**
+ * Deletes an event using its displayed index from the Event Schedule.
+ */
+public class DeleteEventCommand extends Command {
+
+    public static final String COMMAND_WORD = "(ev)delete";
+    public static final String COMMAND_FUNCTION = "Deletes the event identified by the "
+            + "index number used in the displayed event schedule.";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": " + COMMAND_FUNCTION + "\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_DELETE_EVENT_SUCCESS = "Deleted Event: %1$s\nYou now have one less "
```

Comment:
Might want to change the message here cause cause events are not always work related so saying 'Keep up the good work' might be a bit off O: HAHA